download/download.go

```diff
@@ -0,0 +1,181 @@
+package download
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/spf13/afero"
+)
+
+var afs = afero.Afero{Fs: afero.NewOsFs()}
+
+func init() {
+	log.Namespace = "dp-mongodb-in-memory"
```

Comment:
should be set in main ?

Reply:
Being this a library, there is no main function and that's why I decided to use the init method. Is there a better way for this?